changelog.md

```diff
@@ -12,6 +12,7 @@
 ### Units
 ### User interface
 ### WML Engine
+   * Added WFL `byte_index` function to convert between character indices and byte indices, accounting for multi-codepoint characters
```

Comment:
```suggestion
   * Added WFL `byte_index` function to convert between character indices and byte indices, accounting for multi-byte characters
```

&#xC0; is a single codepoint U+C0 which is encoded as 2 bytes in UTF8. `utf8::index("À", 1)` will return 2.
&#x41;&#x300; is two codepoints U+41 and U+300 which is encoded as 3 bytes in UTF8. `utf8::index("À", 1)` will return 1.